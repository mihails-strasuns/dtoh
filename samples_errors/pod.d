module pod;

struct S
{
    this(this)
    {
    }
}

extern(C) void foo(S x);
