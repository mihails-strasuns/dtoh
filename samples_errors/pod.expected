samples_errors/pod.d(3) [pod.S]
	Using non-POD struct as extern(C) is likely to result in hard to debug differences in behaviour
